pkg/splunk/enterprise/indexercluster_test.go

```diff
@@ -262,6 +290,267 @@ func TestIndexerClusterPodManager(t *testing.T) {
 	indexerClusterPodManagerTester(t, method, mockHandlers, 1, splcommon.PhaseScalingDown, statefulSet, wantCalls, nil, statefulSet, pod, pvcList[0], pvcList[1])
 }
 
+func TestSetCmMaintenanceMode(t *testing.T) {
```

Comment:
maybe rename to `TestSetMaintenanceMode` ?

Reply:
Done